aspnetcore/fundamentals/minimal-apis.md

```diff
@@ -1376,6 +1297,44 @@ The following types are bound without explicit attributes:
   app.MapGet("/", (ClaimsPrincipal user) => user.Identity.Name);
   ```
 
+<a name="rbs"></a>
+
+#### Bind the request body as a `Stream` or `PipeReader`
+
+The request body can bind as a [`Stream`](/dotnet/api/system.io.stream) or [`PipeReader`](/dotnet/api/system.io.pipelines.pipereader) to efficiently support scenarios where the user has to process data and:
```

Comment:
This feels like something we want to pull out and expand on under a separate article.